data/human/events.txt

```diff
@@ -136,6 +136,9 @@ event "war begins"
 		"required reputation" 100
 	planet "Martini"
 		description `Martini is a world of exotic beaches and perfect weather. The Republic's central bank and the galactic stock exchange were previously headquartered here, before a terrorist attack destroyed them and a large portion of the capital city as well.`
+		description `	Currently, there is a massive rescue and evacuation operation going on in and around the capital, to find and treat survivors and contain the radiation. As to be expected, almost all supplies are running dangerously low, and Martini's government has called for civilian ships to help deliver goods and supplies for the operation.`
+		spaceport `Instead of the immaculate spaceport that Martini once sported, this temporary spaceport consists of about a hundred hastily poured landing pads, in order to allow the increased numbers of both Navy and civilian ships helping with the rescue and evacuation operation. For whatever reason, someone has hired an entire legion of workers to clean every landing pad before a ship lands, which has caused a considerable queue to form both at the spaceport and in the air. Some habits never change.`
```

Comment:
```suggestion
		spaceport `Instead of the immaculate spaceport that Martini once sported, this temporary spaceport consists of randomly scattered makeshift landing areas, in order to allow the increased numbers of both Navy and civilian ships assisting with the rescue and evacuation operation. For whatever reason, someone has hired an entire legion of workers to clean every landing pad before a ship lands, causing a considerable queue to form both at the spaceport and in the air. Some habits never change.`
```

Reply:
Based off my knowledge of the shit we do in the military, there are places where roads are intentionally long enough and have removable lamposts and other things to facilitate a landing if it's required. It isn't unthinkable to say that the committee in charge of this intentionally used roads as makeshift areas to land.

Also, just a slight thought, but given that the Air Traffic Controller is gone, I'd say the whole operation of landing and such aren't exactly going well.

Reply:
The missions have a spaceport controller that you talk to when you hail Martini... but I am going to say that those are regional spaceports or this temporary spaceport.